components/datetime/src/provider/calendar/symbols.rs

```diff
@@ -34,11 +35,11 @@ pub struct DateSymbolsV1<'data> {
 #[yoke(prove_covariance_manually)]
 pub struct Eras<'data> {
     #[cfg_attr(feature = "provider_serde", serde(borrow))]
-    pub names: ZeroMap<'data, str, str>,
+    pub names: ZeroMap<'data, TinyStr16, str>,
     #[cfg_attr(feature = "provider_serde", serde(borrow))]
-    pub abbr: ZeroMap<'data, str, str>,
+    pub abbr: ZeroMap<'data, TinyStr16, str>,
     #[cfg_attr(feature = "provider_serde", serde(borrow))]
-    pub narrow: ZeroMap<'data, str, str>,
+    pub narrow: ZeroMap<'data, TinyStr16, str>,
```

Comment:
Question: I'm curious why the postcard binary size increases when going to TinyStr16.

Reply:
because most era codes are less than 16 bytes

(on the plus side, lookup is faster, though we've already optimized the hell out of that)